src/main/java/bio/overture/ego/service/RefreshTokenService.java

```diff
@@ -27,18 +31,37 @@
   private RefreshTokenRepository refreshTokenRepository;
 
   /** Configuration */
-  @Value("${refreshToken.duration:10800000}")
-  private int DURATION;
+  private int durationSeconds;
 
   @Autowired
   public RefreshTokenService(
-    @NonNull RefreshTokenRepository refreshTokenRepository,
-    @NonNull UserRepository userRepository,
-    @NonNull UserService userService) {
+      @NonNull RefreshTokenRepository refreshTokenRepository,
+      @NonNull UserRepository userRepository,
+      @NonNull UserService userService,
+      @Value("${refreshToken.durationSeconds:10800000}") int durationSeconds
```

Comment:
does this value refer to what's in the `application.yml`? if it is why does it need the seconds value added? i may be thinking of the yml file incorrectly...i'm looking at the `refreshToken.durationSeconds` as a constant...